frontend/sass/_modal.scss

```diff
@@ -22,21 +22,52 @@
   flex-direction: column;
 }
 
+// This styling applies to desktop Internet Explorer
+.scrollable {
```

Comment:
Did you add this class anew, or is it building on a pre-existing class from Bulma?  If the former, can you prefix this with `jf-`?  That stands for `justfix-`, and we're trying to use it for all our own first-party CSS classes to make it easy to identify which classes are ours, rather than Bulma's.